backend/services/core/src/datasources/BookingRequest.ts

```diff
@@ -176,7 +176,7 @@ export default class BookingRequestAPI extends dbUtils.KnexDataSource {
         title: 'Booking request created',
         message: `${booker.first_name} ${booker.last_name} has created a booking request`,
         link: `/booking?booking=${bookingRequest.id}&endFilter=${bookingRequest.endDate.getTime() + 86_400_000}`, // 24h in ms
-        type: 'BOOKING_REQUEST',
+        type: 'IMPORTANT', // This way it is ALWAYS sent to the KM
```

Comment:
I feel like this should be controlled by KM's settings instead of hard-coded.

Reply:
I thought about this as well, however it would require different user being allowed different notification settings given their different mandates. This was not an easy task, it would be easy if it was an access policy (perhaps "notifications:booking_requests"), before over-engineering it I spoke with Julia who said that she would never want to turn off these notifications.

The idea behind "IMPORTANT" is not specifically "very important notifications" per se, but instead notifications which a user should always receive. The ONLY use case right now is booking requests for KM. 

If we want to have a real priority system for notifications in the future, we might want to take another look at this but I think that this is a fine solution in the meantime (even though "IMPORTANT" is perhaps not the most well-chosen name)

Reply:
Ok, might not be worth the trouble but it seems odd to have hard-coded notification settings. Could show all settings regardless if they have any effect. Or mandate-specific notification settings.